ci-operator/step-registry/assisted/tools/bump-ocp/assisted-tools-bump-ocp-commands.sh

```diff
@@ -6,6 +6,9 @@ set -o pipefail
 
 echo "************ assisted tools bump ocp command ************"
 
+echo "This infrastructure is deprecated. Please refer to https://gitlab.cee.redhat.com/assisted-installer-ops/assisted-installer-cicd/-/tree/main/src/assisted_installer_cicd/bump_ocp_upstream"
```

Comment:
why we need to keep it?

Reply:
just to prevent people adding functionality to the old script at openshift-assisted/assisted-installer-deployment
but alternatively we can just remove that script from there before this PR gets in and make sure merge this right after

Reply:
ah ok, let's merge this now and do the proper cleanup later

Reply:
Implemented at https://github.com/openshift-assisted/assisted-installer-deployment/pull/307

Reply:
we can remove all the files in `ci-operator/step-registry/assisted/tools/bump-ocp/*`, no?

Reply:
not yet. it's registered as a cluster-bot workflow and I'm not sure how fast we'll get the lgtm for it: https://github.com/openshift/release/blob/d8b442d080464217cf48037ddd75c3fc99499b7c/core-services/ci-chat-bot/workflows-config.yaml#L40
I'll follow up with another PR once it's getting in